core/domain/app_feedback_report_domain_test.py

```diff
@@ -600,6 +692,16 @@ def test_validation_invalid_text_input_with_only_text_input_allowed_fails(
             self.user_supplied_feedback,
             'Invalid input text, must be a string')
 
+    def test_validation_empty_selected_items(
```

Comment:
Please make this name clearer.

Reply:
changed `test_validation_empty_selected_items` 
->`test_validation_valid_text_input_with_empty_selected_items`